config/deploy/dev.rb

```diff
@@ -0,0 +1,6 @@
+require_relative 'server_settings'
+
+server 'sdr-services-dev.stanford.edu', user: 'sdr2service', roles: %w{app}
+set :deploy_to, "/var/sdr2service/#{ENV['SDR_APP']}"
+
+Capistrano::OneTimeKey.generate_one_time_key!
```

Comment:
Our goal with this change is to make the deployment practices of this project more consistent with the rest of our portfolio. This is important both to make this project more accessible to the whole team and better align with devops reporting requirements.

I believe @eefahy will be sending our more information to the dlss-developers list soon, but if you're interested, we've updated our documentation here: https://github.com/sul-dlss/DeveloperPlaybook/blob/master/best-practices/deployment.md